docs/ddl-of-carbondata.md

```diff
@@ -646,6 +647,69 @@ CarbonData DDL statements are documented here,which includes:
 
 ## TABLE MANAGEMENT  
 
+### DESCRIBE COMMAND
+
+Describe column of table and visualize its structure with child fields.
+  ```
+  [DESCRIBE | DESC] COLUMN fieldname[.nestedFieldNames] ON [db_name.]table_name;
+
+  Examples: 
+  
+    DESCRIBE COLUMN locationinfo on complexcarbontable;
+    
+    +------------------------------+--------------------------------------------------------------------------------------------------------------------+-------+
+    |col_name                      |data_type                                                                                                           |comment|
+    +------------------------------+--------------------------------------------------------------------------------------------------------------------+-------+
+    |locationinfo                  |array                                                                                                               |null   |
+    |## Children of locationinfo:  |                                                                                                                    |       |
+    |item                          |struct<activeprovince:string,activecity:map<string,string>,activedistrict:string,activestreet:array<string>>        |null   |
+    +------------------------------+--------------------------------------------------------------------------------------------------------------------+-------+
+    
+    DESCRIBE COLUMN locationinfo.item on complexcarbontable
+    
+    +-----------------------------------+------------------+-------+
+    |col_name                           |data_type         |comment|
+    +-----------------------------------+------------------+-------+
+    |locationinfo.item                  |struct            |null   |
```

Comment:
comment section always null ?

Reply:
updated the example where the comment is not null.